postgres_exporter/common/pg16/queries_pg_stat_statements.yml

```diff
@@ -0,0 +1,169 @@
+###
+#
+# Begin File: PG16 queries_pg_stat_statements.yml
+#
+# Copyright © 2017-2023 Crunchy Data Solutions, Inc. All Rights Reserved.
+#
+###
+
+ccp_pg_stat_statements_total:
+  query: "SELECT pg_get_userbyid(s.userid) as role,
+              d.datname AS dbname,
+              sum(s.calls) AS calls_count,
+              sum(s.total_exec_time) AS exec_time_ms,
+              avg(s.mean_exec_time) AS mean_exec_time_ms,
```

Comment:
This is more a critique of this query, and if this is already basically the same as in other pg versions I'll let it slide, but seems like this isn't necessarily a sensible metric, since it's not the mean of the *weighted* average of queries; I'd assume you'd really need to scale each `mean_exec_time` by the number of calls (or use total_exec_time again) and then divide by the sum of the number of calls total) to get the true value here.  Something to think about, and maybe what we have here is "good enough" for most use cases (really seems like only very skewed query workloads would be drastically affected here).

Reply:
The files for PG16 are exactly the same as the ones for 15.

Would certainly welcome critique of the query to improve it in general in another issue.

Reply:
https://github.com/CrunchyData/pgmonitor/issues/368